packages/grid/data-grid/src/tests/toolbar.DataGrid.test.tsx

```diff
@@ -42,65 +50,125 @@ describe('<DataGrid /> - Toolbar', () => {
     }
   });
 
-  it('should increase grid density when selecting compact density', () => {
-    const rowHeight = 30;
-    const { getByText } = render(
-      <div style={{ width: 300, height: 300 }}>
-        <DataGrid {...baselineProps} showToolbar rowHeight={rowHeight} />
-      </div>,
-    );
+  describe('<DataGrid /> - Density selector', () => {
```

Comment:
Already in the parent context

```suggestion
  describe('Density selector', () => {
```